### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by SC7639** [fholger/vrperfkit#142]

**Comment by DigitalFriends** [comment]
Hope not, waiting eagerly for the planned features!

**Comment by SC7639** [comment]
Is direct x 12 support on the roadmap?

**Comment by fholger** [comment]
It is on hold, at the very least, while I'm working on Half-Life 2 VR. I don't have the time to work on it in parallel.

That being said, I don't know if I will do any major development even after HL2VR has quieted down. There is not a whole lot I can realistically improve upon. The majority of issues are game- or headset-specific for games and hardware I do not own.

DX12 or Vulkan support is not planned because it is much more tedious to implement vrperfkit's features for these APIs, and the number of applicable games is just too small. While future games are certainly more likely to feature either of those APIs, they are also much more likely to be built upon OpenXR, which is already covered by the OpenXR toolkit and thus outside of vrperfkit's scope.

**Comment by simon-kuzin** [comment]
This is sad, as vrperfkit was a lifesaver for a number of games.

@fholger 
I would be happy to invest some time to supporting the project. 
Being quite 20+ years software developer, unfortunately  have 0 experience in game/GPU development.
So if you could allocate some  of your time for a quick jump-start, I hopefully will be able at least do some support.

**Comment by shazam25384** [comment]
How well would the project with with dxvk? They ship it with a dx11.dll as well as a dxgi.dll. I'm not sure if vr_perfkit could simply be linked against dvxk libraries instead of DirectX to utilize Vulkan.

**Comment by shazam25384** [comment]
Ok, browsing your folders, I already find the references to dxvk ...
